internal/console/scan.go

```diff
@@ -527,6 +520,22 @@ func createService(inspector *engine.Inspector,
 	return services, nil
 }
 
+func extractPaths(paths []string) (extectedPaths []string, pathExtractionMap map[string]string, err error) {
```

Comment:
I think this would make more sense to be on a source provider, maybe we should create a source provider for zip files, what do you think?